test/components/namespace_tree_component_test.rb

```diff
@@ -1,22 +1,25 @@
 # frozen_string_literal: true
 
 require 'test_helper'
+require 'minitest/autorun'
 
 class NamespaceTreeComponentTest < ViewComponent::TestCase
   test 'Should render a collapsed tree' do
     namespaces = Array.new(3) { |i| Group.create!(name: "Group GLT #{i + 1}", path: "group-glt-#{i + 1}") }
     namespaces.push(Group.create!(name: 'Group GLT 4', path: 'group-glt-4',
                                   children: [Group.create!(name: 'Group GLT 5', path: 'group-glt-5')]))
-    render_inline NamespaceTreeContainerComponent.new(namespaces:, path: 'dashboard_groups_path')
+    Member.stub :effective_access_level, Member::AccessLevel::OWNER do
+      render_inline NamespaceTreeContainerComponent.new(namespaces:, path: 'dashboard_groups_path')
 
-    assert_selector 'li', count: 4
-    assert_selector 'li', text: 'Group GLT 1'
-    assert_selector 'li a[href="/group-glt-1"][aria-label="Group GLT 1"]', count: 1
-    assert_selector 'li', text: 'Group GLT 2'
-    assert_selector 'li a[href="/group-glt-2"][aria-label="Group GLT 2"]', count: 1
-    assert_selector 'li', text: 'Group GLT 3'
```

Comment:
**suggestion:** Can you add a check to make sure the effective role pill is added to the UI?

Reply:
added in abcb534a

Reply:
Looks good, thanks